cmd/cli/license/inspect.go

```diff
@@ -0,0 +1,207 @@
+package license
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/jedib0t/go-pretty/v6/table"
+	"github.com/spf13/cobra"
+
+	"github.com/bacalhau-project/bacalhau/cmd/util/flags/cliflags"
+	"github.com/bacalhau-project/bacalhau/cmd/util/hook"
+	"github.com/bacalhau-project/bacalhau/cmd/util/output"
+	licensepkg "github.com/bacalhau-project/bacalhau/pkg/lib/license"
+)
+
+type InspectOptions struct {
+	LicenseFile string
+	OutputOpts  output.OutputOptions
+}
+
+func NewInspectOptions() *InspectOptions {
+	return &InspectOptions{
+		OutputOpts: output.OutputOptions{Format: output.TableFormat},
+	}
+}
+
+func NewInspectCmd() *cobra.Command {
+	o := NewInspectOptions()
+	cmd := &cobra.Command{
+		Use:           "inspect",
+		Short:         "Inspect license information",
+		Args:          cobra.NoArgs,
+		PreRun:        hook.ApplyPorcelainLogLevel,
+		SilenceUsage:  true,
+		SilenceErrors: true,
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			// Check if license file path is empty or just whitespace
+			if o.LicenseFile == "" || len(strings.TrimSpace(o.LicenseFile)) == 0 {
+				return fmt.Errorf("required flag \"license-file\" not set")
+			}
+
+			// Check if license file exists
+			if _, err := os.Stat(o.LicenseFile); os.IsNotExist(err) {
+				return fmt.Errorf("file not found: %s", o.LicenseFile)
+			}
+			return nil
+		},
+		RunE: func(cmd *cobra.Command, _ []string) error {
+			return o.Run(cmd.Context(), cmd)
+		},
+	}
+
+	// Add flags
+	cmd.Flags().StringVar(&o.LicenseFile, "license-file", "", "Path to the license file")
+	cmd.Flags().AddFlagSet(cliflags.OutputFormatFlags(&o.OutputOpts))
```

Comment:
what can we do to improve the experience? can we just do `bacalhau license inspect <path>` without a dedicated flag? if not, lets introduce short flag version, such as `-f` or `-p`

Reply:
@wdbaruni I agree with adding a short flag `-f`.

Regarding changing it to have no flag for the license file all together, I am also in favor of consistency with other bacalhau commands. I have seen both in other commands, so I went with the more explicit style.

Let me know which one is more consistent with other commands, and I'll go with it.

Reply:
Done: `bacalhau license inspect <path>`